### PR TITLE
Make a recurring donation in any amount

### PR DESCRIPTION
**Pull request by kumar303**
Fixes https://github.com/mozilla/payments-example/issues/30

### CODE REVIEW

**Comment by kumar303** [comment]
man I suck at jquery now. I forgot how to do everything.

File: public/css/index.css [hunk 1]
```diff
@@ -44,3 +44,7 @@ form.button {
 h4 {
     padding-top: 2em;
 }
+
+.donation-variable .form-group .btn {
```
Comment (tofumatt) [inline public/css/index.css:48]: Is `.btn` used by a CSS framework or is that our own convention? If neither I'd opt for `button` over `btn`, we have the bytes to spare :wink:

File: public/js/index.js [hunk 2]
```diff
@@ -147,10 +147,11 @@
     });
   });
 
-  $('form.donation-variable').on('submit', function(event) {
+  $('button.donation-variable').on('click', function(event) {
     event.preventDefault();
+    var productId = $(this).data('product-id');
```
Comment (andymckay) [inline public/js/index.js:152]: it annoys me in jquery that both `data('productId')` and `data('product-id')` work, completely unrelated to this though

**Comment by andymckay** [comment]
r+

File: public/css/index.css (hunk 1 above)
Comment (muffinresearch) [inline public/css/index.css:48]: That's Bootstrap I think.

File: public/js/index.js (hunk 2 above)
Comment (muffinresearch) [inline public/js/index.js:152]: I didn't know about the latter. It used to only work camel-cased. Maybe it now supports both for backwards compatibility.

Thinking back and reading the docs; before HTML5 data attrs, `data()` was also used for attaching and retrieving arbitrary data on a node. This probably explains the camel-case key.

Alternatively you could use attr directly - e.g. `$node.attr('data-foo-bar')`. if you're just reading what's in the markup. Note: this probably misses out on some of the type massaging that `data()` provides (e.g. JSON strings are parsed when using `data()`) .

File: public/css/index.css (hunk 1 above)
Comment (tofumatt) [inline public/css/index.css:48]: As yes, it does that. Okay, ignore me :smile:

File: public/js/index.js (hunk 2 above)
Comment (muffinresearch) [inline public/js/index.js:152]: Oh and also apparently if you use `$foo.data()` to get all the data attrs only the camel-case keys will exist not hyphenated. See http://jsbin.com/dakibofuxa/edit?html,js,console

**Pull request merged**